jwt.go

```diff
@@ -80,10 +81,36 @@ func (t *Token) SigningString() (string, error) {
 	return strings.Join(parts, "."), nil
 }
 
+func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	claim := make(MapClaim)
+	return ParseInterface(tokenString, keyFunc, &claim)
+}
+
+type MapClaim map[string]interface{}
+
+func (m MapClaim) ExpiresAt() (int64, error) {
+	if exp, ok := m["exp"].(float64); ok {
+		return int64(exp), nil
+	}
+	return 0, errors.New("expiration not found")
```

Comment:
Good point, but the way that the validation is done current error here means "do not check the expiration". Maybe if it returned `(int64, bool)` it would be more clear.